thememzking_collector.py

```diff
@@ -0,0 +1,26 @@
+import discord
+import pandas as pd
```

Comment:
Unused import?

Reply:
Yes I first used pandas but removed it without removing the import. Forgot that I didn't need to import discord.

Reply:
I have to use import discord to get new messages. I updated the file to remove the pandas import and the awaiting. Unsure on how to update my pull request :P